server/handlers/changefeed.go

```diff
@@ -0,0 +1,82 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"strings"
+
+	ctxu "github.com/docker/distribution/context"
```

Comment:
Ah, actually when I went and looked at it, this is in the right place. We've been nominally (and we should document it somewhere) stealing Python's PEP8 import formatting, which requires imports be broken into 3 blocks of:
```
import (
    // Std library imports
    "os" // etc...

    // 3rd party dependencies
    "github.com/sirupsen/logrus" // etc...

    // dependencies within the same project
    "github.com/docker/notary" // etc...
)
```